cpu/stm32/Makefile.features

```diff
@@ -3,13 +3,16 @@ include $(RIOTCPU)/stm32/stm32_info.mk
 FEATURES_PROVIDED += bootloader_stm32
 FEATURES_PROVIDED += cpu_stm32$(CPU_FAM)
 FEATURES_PROVIDED += periph_cpuid
-FEATURES_PROVIDED += periph_gpio periph_gpio_irq
+FEATURES_PROVIDED += periph_gpio
+ifneq (g0,$(CPU_FAM))
+  FEATURES_PROVIDED += periph_gpio_irq
```

Comment:
Should at least add a TODO here, so it's clear that this is a bug and not a hardware limitation.

Reply:
I'll try to find the fix, maybe next week. I'd really like to have this family in but not without gpio irq.